frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -431,7 +378,6 @@ function QueryBuilder(props) {
         recentlySaved={uiControls.recentlySaved}
         onOpenModal={openModal}
         onCloseModal={closeModal}
-        onSetRecentlySaved={setRecentlySaved}
```

Comment:
this isn't being used so removing the `setRecentlySaved` element makes the refactor easier